docs/pages/guides/using-bun.mdx

```diff
@@ -31,7 +33,9 @@ EAS decides which package manager to use based on the lockfile in your codebase.
 
 ### Customize Bun version on EAS
 
-EAS uses `bun@1.0.14` by default. If you need to use a particular version of Bun, you can configure the exact version in each build in your **eas.json**:
+Bun is installed by default when using EAS. To check the default version of Bun when creating an EAS Build, see [Android latest server image](/build-reference/infrastructure/#details) and [iOS latest server image](/build-reference/infrastructure/#details-7).
```

Comment:
I'm not sure we should link to the `latest` image. I mean most of the people don't use `latest`. I would rather link to the list of images (https://docs.expo.dev/build-reference/infrastructure/#android-server-images + https://docs.expo.dev/build-reference/infrastructure/#ios-server-images) and let people figure out which image they use and see the image details to check the Bun version installed on the image.